[CI][Navi21] Enable Navi21 codepath for PR and Nightly CI

zhanglx13 opened the pull request:
Three navi workstations are added as jenkins agents. They have the label `gfx1030w && mlir`. 
This PR only enabled navi nodes for PR and Nightly CI only.

zhanglx13 committed:
Enable navi21 codepath

krzysz00 reviewed (commented):
Do you want this approved so you don't have to hound anyone for a review once it's working?

And, since we have one Navi node, how about only running Navi CI nightly and weekly?

zhanglx13 commented:
Yeah, please approve so that I can merge it as long as the node is fully tested.
I'll test only the navi21 codepath for nightly and weekly so that other codepaths will not be too occupied.

krzysz00 commented:
I more meant "don't run Navi21 tests on PRs, even after this lands, because we don't have enough nodes for that"

zhanglx13 committed:
Enable navi21 workstations and jenkins nodes for PR CI only

zhanglx13 commented:
@krzysz00 So now we have three tested navi21 nodes (two workstations and one server node) that are working at the PR CI stage. And since the scope of this PR is changed to PR CI only, I think it is ready to merge.

krzysz00 reviewed (approved):
Seems reasonable to me, approved

zhanglx13 committed:
[CI][Navi21] Enable Navi21 codepath for nightly CI (#835)

Enable navi21 codepath for nightly CI

We limit the number of lit workers to 8 and see how far the navi21
agents can go.
All navi workstations work for all nightly tests. However, the server
nodes only work for the PR CI test. When they are fixed, they can also
be added for nightly CI.
